artifactory/utils/buildinfoproperties.go

```diff
@@ -164,19 +164,20 @@ func ReadGradleConfig(path string, useWrapperIfMissingConfig bool) (config *vipe
 	return
 }
 
-func ReadMavenConfig(path string) (config *viper.Viper, err error) {
+func ReadMavenConfig(path string, useWrapperIfMissingConfig bool) (config *viper.Viper, err error) {
 	if path == "" {
-		config = createDefaultMavenConfig()
+		config = createDefaultMavenConfig(useWrapperIfMissingConfig)
 	} else {
 		config, err = ReadConfigFile(path, YAML)
 	}
 	return
 }
 
-func createDefaultMavenConfig() *viper.Viper {
+func createDefaultMavenConfig(useWrapperIfMissingConfig bool) *viper.Viper {
 	vConfig := viper.New()
 	vConfig.SetConfigType(string(YAML))
 	vConfig.Set("type", Maven.String())
+	vConfig.Set("usewrapper", useWrapperIfMissingConfig)
```

Comment:
noticed that the key you use above is "usewrapper". Shouldn't it be "useWrapper" as you used here?
<img width="622" alt="image" src="https://user-images.githubusercontent.com/7830056/222740167-40274a12-ca57-4541-8d12-ff1ea22945c3.png">

Reply:
Done